src/python/py/models/builder.py

```diff
@@ -239,6 +244,8 @@ def make_genai_config(self, model_name_or_path, extra_kwargs, out_dir):
 
         if self.ep != "cpu":
             ep_options = { self.ep : self.ep_attrs[self.ep] }
+            if self.enable_cuda_graph and self.ep == "cuda":
```

Comment:
If `enable_cuda_graph` is defined in `self.ep_attrs["cuda"]`, this logic should get automatically added to `genai_config` and won't be needed.